@here/harp-mapview/lib/text/Placement.ts

```diff
@@ -841,9 +834,6 @@ export function placePathLabel(
         } else {
             collisionBox.copy(glyphBox);
         }
-        if (checkGlyphVisible && !screenCollisions.isVisible(collisionBox)) {
-            return PlacementResult.Invisible;
-        }
```

Comment:
same as above.

Reply:
I think this check was wrong. Instead of returning invisible if a glyph is not visible, what we should do is skip the collision test for  that glyph that we do just after this.

Reply:
This code has been reverted. A new commit is ready soon.

Reply:
outdated